### PR TITLE
UI Tweaks Part 1

### PR DESCRIPTION
**Pull request by KennethMetz**
1. Prevents watchlist tiles from showing list have "0 items" during the loading state.
2. Display `NoResultsImage` on DetailedView page for empty watchlists.
3. Hides text showing the users handle on `DropMenu` for guests (who don't have a handle)
4. Prevents multiple `AddToListDropMenu` components from being opened at the same time.
5. On the landing page, left aligns the icons on the bottom (3) cards (to match the smaller text).
6. Styles tooltips on `ScoreBars` to have an arrow and appear closer to the component they're meant for.

### CODE REVIEW

File: src/Components/WatchlistTile.js [hunk 1]
```diff
@@ -53,15 +53,17 @@ export default function WatchlistTile({
...
-          </Typography>
+          {items && (
+            <Typography
+              variant="body2"
+              sx={{
+                marginLeft: "8px",
+                flexShrink: 0,
+              }}
+            >
+              {items?.length ?? "0"} {items?.length === 1 ? "item" : "items"}
```
Comment (metzsg) [inline src/Components/WatchlistTile.js:64]: nit: `items` should now be guaranteed to have a truthy value here, so the `?` in `items?` are unneeded

File: src/Components/ProfileMainPage.js [hunk 2]
```diff
@@ -95,7 +95,7 @@ export default function ProfileMainPage() {
           ))}
           {!profile?.savedLists?.length && (
             <Grid item xs={12}>
-              <NoResultsImage />
+              <NoResultsImage tile="true" />
```
Comment (metzsg) [inline src/Components/ProfileMainPage.js:98]: nit: `tile={true}`.  the string "true" is also truthy, but not a boolean, so maybe not quite as accurate.

File: src/Components/NoResultsImage.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
 
-export default function NoResultsImage({ noImage }) {
+export default function NoResultsImage({ noImage, tile }) {
   return (
     <div
       style={{
         display: "flex",
         flexDirection: "column",
-        alignItems: "center",
+        alignItems: !tile ? "center" : "flex-start",
```
Comment (metzsg) [inline src/Components/NoResultsImage.js:13]: Looks nice left-aligned!

File: src/Components/AddToListDropMenu.js [hunk 4]
```diff
@@ -154,7 +154,6 @@ export default function AddToListDropMenu({ anime, variant, selected }) {
           onClick={(e) => {
             handleToggle();
             e.preventDefault();
-            e.stopPropagation();
```
Comment (metzsg) [inline src/Components/AddToListDropMenu.js]: Nice fix, and so simple!  I did think it was a little weird to be able to get a bunch of those boxes open at the same time

File: src/Components/LandingPage.js [hunk 5]
```diff
@@ -220,14 +220,14 @@ export default function LandingPage() {
                 style={{
                   display: "flex",
                   alignItems: "center",
-                  margin: "24px 0px 27px",
+                  margin: "24px 0px 27px 27px",
```
Comment (metzsg) [inline src/Components/LandingPage.js:223]: Looks good!

File: src/Components/ScoreBars.js [hunk 6]
```diff
@@ -22,7 +33,11 @@ export default function ScoreBars({ scores }) {
     <>
       {scores.map((score) => (
         <Box key={score.name} sx={{ display: "flex", alignItems: "center" }}>
-          <Tooltip title={score.description}>
+          <Tooltip
+            title={score.description}
+            arrow={true}
```
Comment (metzsg) [inline src/Components/ScoreBars.js:38]: Cool!

File: src/Components/ScoreBars.js [hunk 7]
```diff
@@ -7,6 +7,17 @@ import { useEffect, useState } from "react";
 export default function ScoreBars({ scores }) {
   const [multiplier, setMultiplier] = useState(0);
 
+  const smallerTopMargin = {
+    modifiers: [
```
Comment (metzsg) [inline src/Components/ScoreBars.js:11]: Nice! Didn't know you could do this

**Review by metzsg — APPROVED** [review]
A lot of great improvements in here.  Nice work!

File: src/Components/WatchlistTile.js (hunk 1 above)
Comment (KennethMetz) [reply src/Components/WatchlistTile.js:64]: Great catch!!

**Pull request merged**